### ISSUE TITLE
Anonymous function with return type hint is not refactored as expected

### ISSUE THREAD

**Issue by gsouf** [squizlabs/PHP_CodeSniffer#979]
The following code:

```php
<?php

function () use ($data) : Something
{
    return call_user_func($data);
};
```

Triggers the rule: 

```
error - Opening brace should be on the same line as the declaration (Squiz.Functions.MultiLineFunctionDeclaration.BraceOnNewLine)
```

Then running phpcb will not refactor the code correctly. It will result in this:

```php
<?php

function () use ($data) {
    : Something

    return call_user_func($data);
};
```

Instead of this:

```php
<?php

function () use ($data) : Something {
    return call_user_func($data);
};
```

Note: I didnt test with non anon function

**Comment by gsherwood** [comment]
Fixed now. Was a problem with non-anon functions as well. Thanks for the report.
